frontend/amundsen_application/base/base_table_quality_client.py

```diff
@@ -0,0 +1,19 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from abc import ABCMeta, abstractmethod
+
+
+class BaseTableQuality(metaclass=ABCMeta):
```

Comment:
The naming for this sounds a bit odd, maybe make it BaseTableQualityChecks, ot BaseTableQualityInterface